src/Avalonia.Controls/ContentControl.cs

```diff
@@ -116,19 +116,14 @@ protected virtual bool RegisterContentPresenter(IContentPresenter presenter)
             return false;
         }
 
-        protected virtual void ContentChanged(AvaloniaPropertyChangedEventArgs e)
+        private void ContentChanged(AvaloniaPropertyChangedEventArgs e)
```

Comment:
Yep, if you're aware of any others, please let us know or open a PR!

Reply:
I did a quick search for "protected virtual" and looked at the results. I would say there are at least 5-10 places this needs to be changed. Automation Peers also don't seem to follow this at all. Perhaps this is because a much older API convention is being used for compatibility.